### PR TITLE
rename argo-cloudops.yaml to cello.yaml

### PR DESCRIPTION
**Pull request by szab100**

### CODE REVIEW

**Review by Acepie — COMMENTED** [review]
Can you add something in the changelog saying the default value changed

**Review by Acepie — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Images moved to from argocloudops to celloproj
 * BREAKING cli binary renamed from argo-cloudops to cello
 
+## [0.13] - 2022-05-02
+### Changed
+* POTENTIALLY BREAKING The default config file has been renamed to chello.yaml (when CONFIG env var is NOT specified)
```
Comment (jk-intuit) [inline CHANGELOG.md:16]: chello.yaml -> cello.yaml

**Review by jk-intuit — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (szab100) [reply CHANGELOG.md:16]: thanks, done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * BREAKING cli binary renamed from argo-cloudops to cello
 * Environment variables prefix changed from ARGO_CLOUDOPS to CELLO (backwards compatible)
 
+## [0.13] - 2022-05-02
```
Comment (jk-intuit) [inline CHANGELOG.md:15]: changelog looks outdated. 0.13.0 is already released and the version needs to be semver format.

https://github.com/cello-proj/cello/blob/main/CHANGELOG.md

Comment (szab100) [reply CHANGELOG.md]: done, 0.13.1 (patch) should be fine compared to 0.13.0, eg. this shouldn't be any more breaking than the prev change

**Review by jk-intuit — APPROVED** [review]

**Pull request merged**